.github/actions/install-rake/action.yml

```diff
@@ -0,0 +1,9 @@
+name: Rake installer  
+jobs:
+description: 'Install Rake'
```

Comment:
I don't think this is an accurate description; running `rake install` doesn't install Rake; in fact, it requires Rake to already be installed! Rather, this uses Rake [to install other dependencies](https://github.com/code-dot-org/code-dot-org/blob/staging/lib/rake/install.rake#L34-L65); most notably NPM packages and Ruby gems